### PR TITLE
refactor: use Vite for VRT app

### PR DESCRIPTION
**Pull request by emmenko**
This is a small refactor to continue the evaluation and adoption of Vite, as an application bundler.

NOTE: the files need to be renamed to `.jsx` so that Vite knows how to parse them.

### CODE REVIEW

File: packages/components/icons/src/icons.visualroute.jsx [hunk 1]
```diff
@@ -0,0 +1,157 @@
+import styled from '@emotion/styled';
+import { Switch, Route } from 'react-router-dom';
+import { ThemeProvider } from '@emotion/react';
+import { customProperties } from '@commercetools-uikit/design-system';
+import * as icons from '@commercetools-uikit/icons';
+import InlineSvg from '@commercetools-uikit/icons/inline-svg';
+import Text from '@commercetools-uikit/text';
+import Spacings from '@commercetools-uikit/spacings';
```
Comment (emmenko) [inline packages/components/icons/src/icons.visualroute.jsx:8]: This import was fixed (before it was a relative import).

File: packages/components/icons/src/icons.visualroute.jsx [hunk 2]
```diff
@@ -61,82 +61,80 @@ const renderIcon = (iconName, color, size) => {
...
+          </li>
+        ))}
+        <li>
+          <a href={`${routePath}/theme`}>{`${routePath}/theme`}</a>
+        </li>
+        <li>
+          <a href={`${routePath}/inline-svg`}>{`${routePath}/inline-svg`}</a>
+        </li>
+      </ul>
+    </Route>
```
Comment (emmenko) [inline packages/components/icons/src/icons.visualroute.jsx:78]: I added these links to the subpages as the `/icons` page does not actually render anything.

File: visual-testing-app/package.json [hunk 3]
```diff
@@ -22,11 +22,7 @@
     "react-router-dom": "5.3.0"
   },
   "devDependencies": {
-    "clean-webpack-plugin": "3.0.0",
-    "html-webpack-plugin": "5.5.0",
-    "moment-locales-webpack-plugin": "1.2.0",
-    "serve": "13.0.2",
-    "webpack": "5.69.1",
-    "webpack-cli": "4.9.2"
```
Comment (emmenko) [inline visual-testing-app/package.json:30]: 👋

File: visual-testing-app/src/App.jsx [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+const darkTheme = {
+  colorSurface: 'black',
+  colorSolid: 'white',
+  colorNeutral60: 'rgba(255,255,255,0.60)',
+  colorNeutral: 'rgba(255,255,255,0.60)',
+  colorAccent98: 'rgba(0,0,0,0.98)',
+};
+const pkgComponentsModules = import.meta.globEager(
+  '../../packages/**/*.visualroute.jsx'
+);
```
Comment (emmenko) [inline visual-testing-app/src/App.jsx:15]: This is kind of the equivalent of Webpack's `require.context`

https://vitejs.dev/guide/features.html#glob-import

**Review by CarlosCortizasCT — APPROVED** [review]
lgtm

**Review by kark — APPROVED** [review]

**Pull request merged**